yarn-project/end-to-end/src/e2e_p2p_network.test.ts

```diff
@@ -77,15 +82,31 @@ describe('e2e_p2p_network', () => {
     for (let i = 0; i < NUM_NODES; i++) {
       const account = privateKeyToAccount(`0x${getPrivateKeyFromIndex(i + 1)!.toString('hex')}`);
       await rollup.write.addValidator([account.address]);
+      logger.debug(`Adding ${account.address} as validator`);
     }
 
+    // Remove the initial sequencer from the set!
+    logger.debug(`Removing ${account.address} as validator`);
+    const txHash = await rollup.write.removeValidator([account.address]);
+
+    await deployL1ContractsValues.publicClient.waitForTransactionReceipt({ hash: txHash });
+
     //@note   Now we jump ahead to the next epoch such that the validator committee is picked
     //        INTERVAL MINING: If we are using anvil interval mining this will NOT progress the time!
     //        Which means that the validator set will still be empty! So anyone can propose.
     const slotsInEpoch = await rollup.read.EPOCH_DURATION();
     const timestamp = await rollup.read.getTimestampForSlot([slotsInEpoch]);
     const cheatCodes = new EthCheatCodes(config.l1RpcUrl);
-    await cheatCodes.warp(Number(timestamp));
+    try {
+      await cheatCodes.warp(Number(timestamp));
+    } catch (err) {
+      logger.debug('Warp failed, time already satisfied');
+    }
+
+    // Send and await a tx to make sure we mine a block for the warp to correctly progress.
```

Comment:
☠️

Reply:
If we were to sleep for `ETHEREUM_SLOT_DURATION`, the same effect would be achieved right?